### ISSUE TITLE
"It should be fine."

### ISSUE THREAD

**Issue by HarbMakes** [tsunamayo/Starship-EVO#5182]
### Short Description of the issue:

Phasing through floors when moving at high speeds, blocks flickering between transparent and opaque, extremely poor framerate.

### Steps to reproduce the issue:

If you sprint and jump and fall for more than about 5 units, you may phase through the floor or the planet.

### Environment
- Starship EVO version: 23w29d
- Graphics Card: GeForce 1050ti

### Additional information:

(https://cdn.discordapp.com/attachments/271723356795961345/1132070808991305768/Starship_EVO_2023-07-21_16-48-47.mp4)
(https://cdn.discordapp.com/attachments/271723356795961345/1132070810685804684/Starship_EVO_2023-07-21_16-45-23.mp4)

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/12134936/Player.log)

**Comment by tsunamayo** [comment]
@HarbMakes Hi, thanks for the videos.
Could you tell me your framerate? You can activate the FPS meter in F3 / Performance / Show FPS info
What is the rest of your PC config?
Also where was the second video taken, with the blinking ground - so I can try to reproduce?
Thanks

**Comment by HarbMakes** [comment]
@tsunamayo 
I have it set to run at 30 frames per second, but it runs closer to 15.
The second video was taken in the first room you spawn in on a new game, the jetpack tutorial.

PC Config? You mean hardware or something else?

**Comment by tsunamayo** [comment]
@HarbMakes Yes hardware, CPU, memory, it should not be that low on a 1050ti...

**Comment by HarbMakes** [comment]
I'll copy it from my steam readout.

**Processor**
CPU Vendor: AuthenticAMD
CPU Brand: AMD Ryzen 5 3600 6-Core Processor
CPU Family: 0x17
CPU Model: 0x71
CPU Stepping: 0x0
CPU Type: 0x0
Speed: 3600 MHz
12 logical processors
6 physical processors
Hyper-threading: Supported
FCMOV: Supported

**Video Card**
Driver: NVIDIA GeForce GTX 1050 Ti
DirectX Driver Name: nvldumd.dll
Driver Version: 31.0.15.3640
DirectX Driver Version: 31.0.15.3640
Driver Date: 6 23 2023
OpenGL Version: 4.6
Desktop Color Depth: 32 bits per pixel
Monitor Refresh Rate: 59 Hz
DirectX Card: NVIDIA GeForce GTX 1050 Ti

**Comment by tsunamayo** [comment]
@HarbMakes Is it a laptop? If yes is it throttling? What kind of FPS are you doing in other games?
Also last point, is the resolution of the video you one you play in?
Your PC config is not that bad you should have better FPS.

**Comment by HarbMakes** [comment]
@tsunamayo 
It is a PC. Other games usually run 30 to 45. The resolution is at my ultrawide 2560x1080.

**Comment by tsunamayo** [comment]
@HarbMakes Thanks for the feedback. Could you tell me your FPS while you are in space, looking at the void? I want to check if you are CPU bound or GPU bound.

**Comment by HarbMakes** [comment]
@tsunamayo 
With my face-to-space I'm getting 30.

**Comment by tsunamayo** [comment]
@HarbMakes I realized there is no RAM stats from your config, what is your RAM (just the size, frequency just dont have impact)

**Comment by HarbMakes** [comment]
@tsunamayo 
Oh! Forgot about that. 16 GB
